lua/entities/m9k_oribital_cannon.lua

```diff
@@ -620,7 +620,7 @@ if SERVER then
         smokering:SetScale( 1 ) --otherwise you'll get the pinch thing. just leave it as it is for smoke, i'm trying to save on lua files dammit!
         util.Effect( "m9k_orbital_smokering", smokering )
 
-        for k, v in pairs( player.GetAll() ) do
+        for k, v in player.Iterator() do
```

Comment:
I rather have `ipairs( player.GetAll() )` than `player.Iterator()`
`player.Iterator()` isnt gonna save enough performance here for its unreliability to be worth it